Cannot install package.json from template

I get the following error:
jan-hendrik@JanHendriksAir okapi4 % cd js
jan-hendrik@JanHendriksAir js % npm install
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:^


package.json
```
"dependencies": {
"@anywidget/react": "workspace^",
"react": "^18.2.0",
"react-dom": "^18.2.0"
},
```

--- a/.changeset/short-weeks-care.md
+++ b/.changeset/short-weeks-care.md
@@ -1,0 +1,5 @@
+---
+"create-anywidget": patch
+---
+
+fix: ensure templates have correct @anywidget/\* versions

--- a/packages/create-anywidget/template-react/js/package.json
+++ b/packages/create-anywidget/template-react/js/package.json
@@ -4,7 +4,7 @@
 		"build": "esbuild --minify --format=esm --bundle --outdir=../src/my_widget widget.jsx"
 	},
 	"dependencies": {
-		"@anywidget/react": "workspace:^",
+		"@anywidget/react": "^0.0.2",
 		"react": "^18.2.0",
 		"react-dom": "^18.2.0"
 	},

--- a/packages/create-anywidget/template-vanilla-ts/js/package.json
+++ b/packages/create-anywidget/template-vanilla-ts/js/package.json
@@ -5,7 +5,7 @@
 		"typecheck": "tsc --noEmit"
 	},
 	"devDependencies": {
-		"esbuild": "^0.18.14",
+		"esbuild": "^0.19.2",
 		"typescript": "^5.2.2"
 	},
 	"dependencies": {

